### PR TITLE
UNO-235 move ckeditor, simulator to tao-core-libs-fe

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/UNO-235

`CKeditor` and `simulator` were moved from `tao-core`, `tao-core-ui-fe`,` tao-item-runner-qti-fe` to `tao-core-shared-libs-fe` to reduce duplication of code https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1
How to update CKeditor in `lib/ckeditor/README.md` in https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1

How to test:
`npm run test` or `npx cross-env PARALLEL_TESTS=1 npm test`

**package.json should be updated after merge tao-core-shared-libs-fe:**

- [x] https://github.com/oat-sa/tao-core-shared-libs-fe/pull/1

Related PRs:
https://github.com/oat-sa/extension-tao-devtools/pull/178
https://github.com/oat-sa/tao-item-runner-qti-fe/pull/73
https://github.com/oat-sa/tao-core/pull/2498

### CODE REVIEW

File: test/pageStatus/test.js [hunk 1]
```diff
@@ -81,18 +81,18 @@ define(['lodash', 'jquery', 'ui/pageStatus'], function(_, $, pageStatusFactory)
                 );
 
             _.delay(function() {
-                popup.close();
-            }, 200);
+                popup && popup.close();
+            }, 600);
```
Comment (habemuscode) [inline test/pageStatus/test.js:85]: This change is for some product decisions?

File: test/pageStatus/test.js [hunk 2]
```diff
@@ -81,18 +81,18 @@ define(['lodash', 'jquery', 'ui/pageStatus'], function(_, $, pageStatusFactory)
...
 
             setTimeout(function() {
                 ready();
-            }, 500);
+            }, 900);
```
Comment (habemuscode) [inline test/pageStatus/test.js:89]: This change is for some product decisions?

Comment (bziondik) [reply test/pageStatus/test.js:89]: window.open doesn't open popup immediately, I check locally, and change the delay

Comment (habemuscode) [reply test/pageStatus/test.js:89]: :)

**Comment by habemuscode** [comment]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: build/postcss.config.js [hunk 3]
```diff
@@ -17,12 +17,12 @@
  */
 
 const path = require('path');
```
Comment (btamas) [inline build/postcss.config.js:19]: `path` module is unused.

File: test/themeLoader/test.js [hunk 4]
```diff
@@ -178,7 +178,7 @@ define(['lodash', 'jquery', 'ui/themeLoader'], function(_, $, themeLoader) {
             var $styleSheets = $('link[data-type^="custom-theme"]');
             assert.ok($styleSheets.length > 0, 'The styleSheets have been inserted');
             assert.equal($styleSheets.length, 3, 'All styleSheets have been inserted');
-
+
```
Comment (btamas) [inline test/themeLoader/test.js:181]: Usless whitespaces

File: build/postcss.config.js (hunk 3 above)
Comment (bziondik) [reply build/postcss.config.js]: fixed

File: test/themeLoader/test.js (hunk 4 above)
Comment (bziondik) [reply test/themeLoader/test.js]: fixed

**Comment by bziondik** [comment]
> I do not really like the timeout increases. Maybe, the issue is the speed of your computer. My computer can run the tests without issue and jenkins automated tests could do it. These modifications can extend the test running process for no reason.
> By default the `tao-qunit-testrunner` sets `PARALLEL_TESTS` to 5 (https://github.com/oat-sa/tao-qunit-testrunner-fe#environment-variables)
> In the jenkins file, I had to decrease it to 2, because it has limited resources (https://github.com/oat-sa/tao-core-ui-fe/blob/master/.Jenkinsfile#L15)
> Can you try the following `npx cross-env PARALLEL_TESTS=1 npm test`

Yes, you are right. Revert changes in timeout

**Pull request merged**